### PR TITLE
add deploy configuration to circle.yml

### PR DESCRIPTION
**Pull request by relud**
still needs creds set in circle ci

### CODE REVIEW

**Comment by mythmon** [comment]
I see this in the Circle output

```
$ mkdir -p ~/docker; test '!' -e ~/docker/image.tar || rm ~/docker/image.tar; docker save app:build > ~/docker/image.tar; ls -l ~/docker

Error response from daemon: could not find image: no such id: app:build
total 0
-rw-rw-r-- 1 ubuntu ubuntu 0 Feb 29 21:30 image.tar 
```

I think we need to `s/app:build/normandy:build/g`?

**Comment by relud** [comment]
yes

File: circle.yml [hunk 1]
```diff
@@ -4,10 +4,34 @@ machine:
...
   override:
     - docker run --net host -e DJANGO_CONFIGURATION=Test normandy:build py.test
+
+# appropriately tag and push the container to dockerhub
+deployment:
+  latest:
+    branch: master
+    commands:
+      # set DOCKER_DEPLOY=true in Circle UI to do deploys
+      - "${DOCKER_DEPLOY:-false}"
```
Comment (mythmon) [inline circle.yml]: tricky

**Comment by mythmon** [comment]
The test failed this time. I'm not quite sure what happened.

Do you want to hack on the cache stuff more? Or we can leave that for another PR if you'd prefer. The tests are still quite fast without the cache.

**Comment by relud** [comment]
nah, I fixed it. I've also added the secrets to circle

**Comment by mythmon** [comment]
Nice.

**Pull request merged**